src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildManager.kt

```diff
@@ -270,10 +268,9 @@ object CargoBuildManager {
         notification.notify(project)
 
         if (messageType === MessageType.ERROR) {
-            MessageView.SERVICE.getInstance(project) // register ToolWindowId.MESSAGES_WINDOW
```

Comment:
Not sure that we don't need call `BuildContentManager.getInstance(project).getOrCreateToolWindow()` here instead.
@mchernyavsky why did we call it here before?

Reply:
I'm not sure. It is copypasted from `CidrBuild.showBuildNotification` :)